lib/shape.ml

```diff
@@ -9,6 +9,16 @@ type shapes = shape list
 
 let canvas_mid = { x = 250; y = 250}
 
+let axes_flag = ref false
+let draw_axes flag = 
+  axes_flag := flag
+
+let render_axes () = 
+  moveto (size_x () / 2) 0;
+  lineto (size_x () / 2) (size_y ());
+  moveto 0 (size_y () / 2);
+  lineto (size_x ()) (size_y () / 2);;
```

Comment:
Double `;;` is not necessary in source files. It's useful when we're running code in a REPL, like `utop`.

Reply:
That's what I thought as well, but this was the only way I could get it to compile without throwing the syntax error I had mentioned in the issue.

Reply:
That's odd. I think it should work without the semi-colons as well. But no issues keeping it as-is.

Reply:
Fixed, it needed to have no semicolon at all in the last line of the function. Also moved the set_color call to 'render_axes' because I think that makes more sense.